changelog/2023-07-19-conduit-0-7-0-release.md

```diff
@@ -0,0 +1,19 @@
+---
+slug: '2023-07-19-conduit-0-7-0-release'
+title: Conduit 0.7.0 release
+draft: false
+tags: [conduit, release]
+---
+
+[**Conduit v0.7**](https://github.com/ConduitIO/conduit/releases/tag/v0.7.0) has a new release! We continue to enhance Conduit as a powerful alternative to Kafka Connect, moving closer to our goal of supporting elaborate data pipelines.
+
+<!--truncate-->
+
+### Key Highlights
+
+- **Native Schema Registry Support**: This release introduces native schema registry support, allowing you to store metadata about the data flowing through your pipelines. This ensures data types and required fields are enforced, enhancing confidence in the data being processed. You can interact with the schema registry using four built-in processors: Decode with Schema Key, Decode with Schema Payload, Encode with Schema Key, and Encode with Schema Payload. Currently, only Avro is supported, with plans to add Protobuf and JSON schema in future releases.
```

Comment:
The first part is a bit generic and ambiguous. 0.7 introduced the Avro processors, which make it possible to encode and decode records using an existing Confluent Schema Registry.

Reply:
What do you think of this?

-  **Native Schema Registry Support**: This release enhances Conduit with native support for the Confluent Schema Registry, enabling you to encode and decode records using Avro schemas. This functionality allows you to store metadata about the data flowing through your pipelines, ensuring that data types and required fields are enforced, which boosts confidence in the integrity of the processed data. You can interact with the schema registry through four built-in processors: Decode with Schema Key, Decode with Schema Payload, Encode with Schema Key, and Encode with Schema Payload. Future updates will include support for additional formats like Protobuf and JSON schema.

Reply:
This reads very good :+1: .